README.md

```diff
@@ -1 +1,2 @@
 # SGNL ingestion adapter framework
+Please see https://eng.playbooks.sgnl.host/playbooks/adapters/development/ for a detailed guide on writing adapters.
```

Comment:
From my understanding these playbooks aren't intended to be public facing - should we be linking to these from a public repo that is intended to be shared with customers, etc?